install/terraform/open-match-build/open-match-build.tf

```diff
@@ -32,6 +32,25 @@ variable "vpc_flow_logs" {
   default     = "false"
 }
 
+variable "workload_identity_account_namespace" {
+  description = "Kubernetes namespace where the workload identity service account will take effect"
+  default     = "open-match"
+}
+
+# Defines the name of the Kubernetes service account used to auto upload the stress test result. 
+# Change kubernetes.serviceAccount for install/helm/open-match/subcharts/open-match-test/values.yaml 
+# file if you want to modify this name.
+variable "workload_identity_account_ksa" {
+  description = "Name of the Kubernetes service account which binds to the Google service account"
+  default     = "stress-test-upload-ksa"
+}
+
+# Defines the name of the Google service account used to bind with the KSA in open-match-test subchart.
+variable "workload_identity_account_gsa" {
+  description = "Name of the Google service account which binds to the Kubernetes service account"
+  default     = "stress-test-upload-gsa"
```

Comment:
gsa is redundant here.

Reply:
The reason that I have gsa and ksa here is to distinguish these two types of service account. Otherwise people who wanna modify this file may spend a lot of time to figure out what they represent